modules/openapi-generator/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -27,9 +27,9 @@ import java.io.IOException;
 import java.io.InputStream;
 
 import java.net.URI;
-import javax.net.ssl.SSLContext;
```

Comment:
`javax.net.ssl` package ist [unaffected by Jakarta EE](https://github.com/eclipse-ee4j/jakartaee-platform/blob/master/namespace/unaffected-packages.adoc).
This change should be reverted.

Reply:
thanks for spotting. I've create PR for fix this https://github.com/OpenAPITools/openapi-generator/pull/14341